virt/release_notes/virt-4-19-release-notes.adoc

```diff
@@ -307,4 +307,99 @@ You can also disable free page reporting of memory ballooning for all new VMs. T
 // cnv-56890 - threads
 * In the {product-title} web console, it is erroneously possible to define multiple CPU threads for a VM based on s390x architecture. If you define multiple CPU threads, the VM enters a `CrashLoopBackOff` state with the `qemu-kvm: S390 does not support more than 1 threads` error. (link:https://issues.redhat.com/browse/CNV-56890[CNV-56890])
 
+[id="virt-4.19-asynch-releases_{context}"]
+== Asynchronous releases
+
+Release notes for asynchronous releases of Red Hat {VirtProductName}.
+
+[id="virt-4.19.1_{context}"]
+=== 4.19.1
+
+.New and changed features
+
+//CNV-57758 DOC: deploying clusters with GPFS
+* Red Hat {VirtProductName} now enables users to deploy VMs on a scalable, clustered file system. {IBMFusionFirst} offers access to consolidated, block-level data storage. It presents storage devices, such as disk arrays and tape libraries, to the operating system as if they were direct-attached storage.
++
+The {FusionSAN} Operator is available in the {product-title} Operator hub.
++
+See xref:../../virt/fusion_access_SAN/virt-about-fusion-access-san.adoc
+#about-fusion-access-san[About {FusionSAN}] for more information.
+
+.Known issues
+
+//OCPNAS-56 Failed localdisk should have its status reflected 
+* When a file system in {FusionSAN} has 2 local disks and one fails, both local disks move to the `Unknown` state, with no indication which of the local disks failed.
+//(link:https://issues.redhat.com/browse/OCPNAS-56[OCPNAS-56])
+
+//OCPNAS-61 Removing primary filesystem causes GPFS storage to become unusable
+* When creating more than one file system for VM storage in {FusionSAN}, deleting the initial, primary file system results in all of the remaining file systems becoming unusable. You can not migrate or restart any of the VMs running on the remaining file systems, and you can not create new VMs on the remaining file systems.
```

Comment:
Can we also add a snippet for how to determine which file system is the primary?
```
$ oc get cso -n ibm-spectrum-scale-csi ibm-spectrum-scale-csi -o jsonpath='{.spec.clusters[*].primary.primaryFs}'
```